### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jobinthomasnypl**
updated documentation to reflect @nypl/sierra-wrapper.

### CODE REVIEW

**Comment by jobinthomasnypl** [comment]
Should we say starting v0.2.0 use `require ('@nypl/sierra-wrapper')` but for versions before that (**we encourage you to update to latest and change client to reflect the new require format**) use `require ('sierra-wrapper')`

**Comment by EdwinGuzman** [comment]
Yes, that's a very good and valid point. Can you include that as well?

**Comment by jobinthomasnypl** [comment]
@EdwinGuzman,

Changes made to the documentation, when you get a chance can you approve this request, if this all that is needed.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ To use make sure you have your credentials stored in a json file in the format:
 You then authorize and request by
 
 ```
+//  starting v0.2.0 use require ('@nypl/sierra-wrapper') but for versions before that (we encourage you to //  update to latest and change client to reflect the new require format) use require ('sierra-wrapper')
```
Comment (EdwinGuzman) [inline README.md]: The formatting is weird here with the second pair of `//`. That should be a new line.

**Comment by jobinthomasnypl** [comment]
@EdwinGuzman fixed the commented line

**Pull request merged**